.gitignore

```diff
@@ -14,3 +14,5 @@ bin/
 
 # Log Files
 *.log
+
+.metadata/
```

Comment:
What IDE are you using to need to exclude `.metadata/`?

Reply:
Intellij Idea.

Reply:
Thought that was just eclipse?  https://gist.github.com/fedir/cd58012d42c8c1edbb3fff611de6c2f6  

![image](https://user-images.githubusercontent.com/55091577/230607877-c2d43407-a151-4b7a-b409-9f547f5a3697.png)

Reply:
I did open this in STS because my Intellij Ultimate trial ran out, then I downloaded Intellij CE and ran the app from the command line.